doc/orchestrator.rst

```diff
@@ -1,208 +1,686 @@
+    :language: python
+    :linenos:
+    :lines: 1-9
+
+Orchestrator Initialization
+---------------------------
+In the next stage of the experiment, we create a standalone ``Orchestrator``.
+
+To create a standalone ``Orchestrator``, utilize the ``Experiment.create_database()`` function:
+
+.. literalinclude:: ../tutorials/doc_examples/orch_examples/std_driver.py
+    :language: python
+    :linenos:
+    :lines: 11-12
+
+Client Initialization
+---------------------
+The SmartRedis ``Client`` object contains functions that manipulate, send, and retrieve
+data on the ``Orchestrator``. Begin by initializing a SmartRedis ``Client`` object for the standalone ``Orchestrator``.
+
+SmartRedis ``Clients`` in driver scripts do not have the ability to use a `db-identifier` or
+rely on automatic configurations to connect to ``Orchestrators``. Therefore, when creating a SmartRedis client
+connection from within a driver script, specify the address of the ``Orchestrator`` you would like to connect to.
+You can easily retrieve the ``Orchestrator`` address using the ``Orchestrator.get_address()`` function:
+
+.. literalinclude:: ../tutorials/doc_examples/orch_examples/std_driver.py
+    :language: python
+    :linenos:
+    :lines: 14-15
+
+Data Storage
+------------
+In the application script, we retrieved a NumPy tensor stored from within the driver script.
+To support the application functionality, we create a
+NumPy array in the experiment driver script to send to the ``Orchestrator``. To
+send a tensor to the ``Orchestrator``, use the function ``Client.put_tensor(name, data)``:
+
+.. literalinclude:: ../tutorials/doc_examples/orch_examples/std_driver.py
+    :language: python
+    :linenos:
+    :lines: 17-20
+
+Model Initialization
+--------------------
+In the next stage of the experiment, we configure and create
+a SmartSim ``Model`` and specify the executable path during ``Model`` creation:
+
+.. literalinclude:: ../tutorials/doc_examples/orch_examples/std_driver.py
+    :language: python
+    :linenos:
+    :lines: 22-27
+
+File Generation
+---------------
+To created an isolated output directory for the ``Orchestrator`` and ``Model``, invoke ``Experiment.generate()`` via the
+``Experiment`` instance `exp` with `standalone_orchestrator` and `model` as input parameters:
+
+.. literalinclude:: ../tutorials/doc_examples/orch_examples/std_driver.py
+    :language: python
+    :linenos:
+    :lines: 29-30
 
+.. note::
+  Invoking ``Experiment.generate(standalone_orchestrator, model)`` will create two directories:
+  `standalone_orchestrator/` and `model/`. Each of these directories will store
+  two output files: a `.out` file and a `.err` file.
 
-To create a colocated model, first, create a ``Model`` instance and then call
-the ``Model.colocate_db_tcp`` or ``Model.colocate_db_uds`` function.
+.. note::
+  It is important to invoke ``Experiment.generate()`` with all ``Experiment`` entity instances as input
```

Comment:
It is important to invoke ``Experiment.generate()`` with all ``Experiment`` entity instances as input
  as well as before any entities have been launched. 
  
  ->
  
  It is important to invoke ``Experiment.generate()`` with all ``Experiment`` entity instances before launching.